### DISCUSSION TITLE
I need help setting up Mastodon using DigitalOcean droplet

### DISCUSSION THREAD

**Discussion by tmonkey** [mastodon/mastodon#19522] [General]
I've gone through the process a few times and got tripped up on different places. I'd really appreciate someone who can help me debug my setup. Here's what I've done so far:
- registered a domain name with Hover; SMTP with sendinblue; S3 bucket
- set up a Digitalocean account and set up the Mastodon droplet
- set up ssh key but when I try to ssh in it says permission denied
- tried using the recovery console
- got through the setup wizard there, set up S3 and SMTP, but got to the final step, which was using letsencrypt to get a SSL cert. Threw an error "nginx: [emerg] BIO_new_file ... failed ... no such file or directory. Not sure why it says nginx is misconfigured.
Halp!

**Comment by Tslmuun** [comment]
https://docs.joinmastodon.org/admin/install/

**Reply by tmonkey** [reply]
Thanks. I just destroyed the old Droplet and started a new one, which worked. It was more like trial and error. But thanks!
